book/src/debug_console.md

```diff
@@ -1,8 +1,16 @@
-# Debug Logging
+# Debug Console
 
-Ariel OS supports debug logging on all platforms. It is enabled by default.
+## Printing on the Debug Console
 
-## Logging
+The debug console is enabled by default and the corresponding [laze module][laze-modules-book] is `debug-console`.
+The [`ariel_os::debug::print!()`][print-macro-rustdoc] macro is used to print on the debug console.
```

Comment:
sorry, maybe also mention `println!()`? Personally I use that in 99% of the cases if not using log.